src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] {
+                new Truck("truck"),
+                new Bulldozer("bulldozer"),
+                new Excavator("excavator")
+        };
+        for (int i = 0; i < machines.length; i++) {
+            machines[i].doWork();
```

Comment:
Ensure that the messages in `doWork` methods are informative and indicate the type of machine that is working. For example, the message for `Truck` might be "Truck is doing its work."